pkg/storage/trb_request_document.go

```diff
@@ -5,57 +5,53 @@ import (
 	"fmt"
 
 	"github.com/google/uuid"
+	"github.com/lib/pq"
 	"go.uber.org/zap"
 
 	"github.com/cms-enterprise/easi-app/pkg/appcontext"
 	"github.com/cms-enterprise/easi-app/pkg/apperrors"
 	"github.com/cms-enterprise/easi-app/pkg/models"
+	"github.com/cms-enterprise/easi-app/pkg/sqlqueries"
 )
 
 // GetTRBRequestDocumentsByRequestID queries the DB for all documents attached to the TRB request with the given ID
 // It will NOT return any items that have been soft-deleted (deleted_at NOT NULL)
 func (s *Store) GetTRBRequestDocumentsByRequestID(ctx context.Context, trbRequestID uuid.UUID) ([]*models.TRBRequestDocument, error) {
```

Comment:
Could we replace this with a call to `GetTRBRequestDocumentsByRequestIDs` with a slice of a single ID? That way we don't have to duplicate any of the SQL / store logic?

Reply:
We could, but I've left these as separate queries in all the PRs so far because I imagine it's more performant to use `WHERE id=:id` instead of `WHERE id=ANY(id_list_of_one_id)` and I didn't really see the benefit of NOT writing the separate query aside from the time it took to write the code.

That said, I could take a second pass later and delete all the single calls if you feel strongly about it.

Reply:
[I have no strong feelings one way or the other!](https://www.youtube.com/watch?v=CxK_nA2iVXw)

I just thought it could be a good change to reduce the number of places we have similar/identical SQL so if we change one we don't have to change the other, but it's a pretty minor smell anyways. Let's leave it as-is for this PR!